app/utils/organisation_name_validation.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const trim = require('lodash/trim')
+const { isEmpty, isFieldGreaterThanMaxLengthChars } = require('../browsered/field-validation-checks')
+const ORGANISATION_NAME_MAX_LENGTH = 50
```

Comment:
I think we agreed on `255` when we discussed last time - https://github.com/alphagov/pay-adminusers/pull/373/files#diff-d10a0b9d3e0a9c446b2bb9cf543bd37eR24
Let me know if there are some updates on this, thanks

Reply:
good spot, i missed this